process/track/baseBlockTrack.go

```diff
@@ -462,6 +450,11 @@ func (bbt *baseBlockTrack) getFinalHeader(shardID uint32) (data.HeaderHandler, [
 	return bbt.selfNotarizer.GetFirstNotarizedHeader(shardID)
 }
 
+// CheckBlockAgainstWhiteList returns if the provided intercepted data (blocks) is whitelisted or not
+func (bbt *baseBlockTrack) CheckBlockAgainstWhitelist(interceptedData process.InterceptedData) bool {
```

Comment:
A little bit over engineered to have a function that has only one simple instruction that already explains what it does